controller/controller.go

```diff
@@ -24,8 +24,12 @@ func Handle(cfg *config.Config) {
 	if os.Getenv("ENVIRONMENT") == "dev" {
 		router = handler.DevelopmentCORS(router)
 	}
+
+	tlsConfig, _ := cfg.Web.TLSConfig()
+
```

Comment:
```suggestion
	tlsConfig, _ := cfg.Web.TLSConfig()
```
Also, could you check the `err` instead of skipping it?

I would rather fail fast than let the app silently run with a potentially hidden issue.

Reply:
Ok, I added a panic there, because `Handle()` doesn't return an error value. But this error branch is unreachable as long as there is a separate config validation before: https://github.com/TwiN/gatus/blob/5495ff8088eb05d72184f28b4dd14c94291ba0a9/config/web/web.go#L60
So it fails even earlier anyway. I tested it with an invalid certificate file. This produces:
```
❯ ./gatus
2023/04/22 14:06:00 [config][LoadConfiguration] Reading configuration from configFile=
2023/04/22 14:06:00 [config][validateAlertingConfig] Alerting is not configured
2023/04/22 14:06:00 [config][validateEndpointsConfig] Validated 1 endpoints
panic: invalid tls config: tls: failed to parse private key

goroutine 1 [running]:
main.main()
        ~/code/gatus/main.go:19 +0x168
```

Reply:
You're right, my bad. I'll remove it back and add a comment myself - you've already done more than enough work on this and I've dragged the review on for too long at this point 😂